src/components/DateRangeFilter/DateRangeFilter.tsx

```diff
@@ -122,6 +138,9 @@ export const DateRangeFilter = ({ withRevokedRange, className }: DateRangeFilter
         <Button onClick={onApplyDateRange} className="mt-4 w-full" variant="default">
           Apply
         </Button>
+        <Button onClick={onClearDateRange} className="mt-4 w-full" variant="secondary">
+          Clear
+        </Button>
```

Comment:
In the app (with the "filers" box closed) there should be some visual information that the filters are applied. 
> There is no info that some filters are applied.

Reply:
Added info message when filters are applied. What visual information I should add to display that filters are applied when the box is closed?

Reply:
This is example of the ideas:

- Use Clear Labels or Tags:
    - Display active filters as labels or tags above the content list.
    - Allow users to remove filters directly by clicking an 'X' on each tag.
    - Highlight Filter Icons or Buttons:

- Change the color or style of filter icons when they are active.
    - Add a small badge or indicator next to the icon showing the number of active filters.
    - Provide a Summary Bar:

- Include a concise summary that lists all active filters in one line.
    - Ensure it's easily noticeable but doesn't take up too much space.
    - Implement a 'Reset Filters' Option:

- Offer a single button to clear all filters, simplifying the reset process.
    - Place it where users expect to find it, such as near the filter options or at the top of the content area.
    - Avoid Overloading the Interface:

Reply:
Added Icon